packages/expo-contacts/src/Contacts.ts

```diff
@@ -943,6 +943,18 @@ export async function getGroupsAsync(groupQuery: GroupQuery): Promise<Group[]> {
   return await ExpoContacts.getGroupsAsync(groupQuery);
 }
 
+/**
+ * Presents a native contact picker to select a single contact from the system. On iOS, no permissions are required to use this method. On Android, the `READ_CONTACTS` permission is required. You can
+ * obtain this permission by calling the [Contacts.requestPermissionsAsync()](#contactsrequestpermissionsasync) method.
```

Comment:
```suggestion
 * Presents a native contact picker to select a single contact from the system. On Android, the `READ_CONTACTS` permission is required. You can
 * obtain this permission by calling the [Contacts.requestPermissionsAsync()](#contactsrequestpermissionsasync) method. On iOS, no permissions are 
 * required to use this method. 
```

Let's use the correct platform order [as per our writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-android-ios-and-web).